winrm4j/pom.xml

```diff
@@ -40,11 +40,5 @@
       <artifactId>winrmj4-client</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <version>${guava.version}</version>
-      <scope>test</scope>
```

Comment:
Let's keep this a test dependency. No reason to force guava on users of the library. What's the reason to move it to root?